### DISCUSSION TITLE
Starting lists from >1.

### DISCUSSION THREAD

**Discussion by JanAlbr-FhG** [vslinko/obsidian-outliner#408] [Archive]
Thank you for your work.

I often like to insert newlines in a list containing text code-blocks. I then want to resume the list after the content. The problem i have is, that outliner then starts a new list from 1.

1. Foo
2. Bar
```C
printf("foobar!");
```
1. THIS SHOULD BE 3.

I can circumvent this when i finish the list first, and then shift-enter between the entries. But that leads to some weird alignment plus i may forget the content i wanted to add.

So my feature suggestion is allowing lists to start from values >1. .
Thank you, kind regards

**Comment by vslinko** [comment]
@JanAlbr-FhG Hi!

You need to indent code block to achive that: [example](https://markdown-it.github.io/#md3=%7B%22source%22%3A%221.%20Foo%5Cn2.%20Bar%5Cn%20%20%20%20%60%60%60C%5Cn%20%20%20%20printf%28%5C%22foobar!%5C%22%29%3B%5Cn%20%20%20%20%60%60%60%5Cn1.%20THIS%20SHOULD%20BE%203.%22%2C%22defaults%22%3A%7B%22html%22%3Afalse%2C%22xhtmlOut%22%3Afalse%2C%22breaks%22%3Afalse%2C%22langPrefix%22%3A%22language-%22%2C%22linkify%22%3Atrue%2C%22typographer%22%3Atrue%2C%22_highlight%22%3Atrue%2C%22_strict%22%3Afalse%2C%22_view%22%3A%22html%22%7D%7D)
